auth/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.13.7-alpine
+
+WORKDIR /auth
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+COPY config.json /etc/auth-service/
+
+RUN go build -o auth
+
+ENTRYPOINT ./auth
+
+EXPOSE 82
```

Comment:
Do we need to be exposing different ports for each service?

Reply:
Good question: have done that here for consistency with the other services. I'll wait until we move to Kube to change it, if it is possible